docs/constitution.rst

```diff
@@ -0,0 +1,60 @@
+Mu's Constitution
+=================
+
+Our Values
+----------
+
+Happily, Mu's users and developers are a diverse and international community
+who maintain a reputation as a friendly, welcoming and dynamic group.
+
+We trust that such collaborators will treat each other in a way that reflects
+the widely help view that diversity and friendliness are strengths of our
+community to be celebrated and fostered.
+
+We believe anyone involved in Mu's community has a right to:
+
+* be treated with courtesy, dignity and respect;
+* be free from any form of discrimination, victimisation, harassment or
+  bullying.
+* enjoy an environment free from unwelcome behaviour, inappropriate language
+  and unsuitable imagery.
+
+To this end, we hope participants in our community abide by the spirit of our
+:doc:`code_of_conduct`.
+
+The Role of Administrators
+--------------------------
+
+Contributing to Mu
+------------------
+
+Managing Expectations
+---------------------
+
```

Comment:
Strawman Suggestion here:

We recognise that Mu is a useful everyday tool in classrooms and coding club situations. And we're delighted that the tool we've built helps educators as much as it does.

However, Mu is no-one's full-time job. No-one earns money from developing Mu and all of us involved offer our time freely as a benefit to the educational coding community. The amount of time we can dedicate to Mu is constrained by Real Life.

Typically you'll see some development work on Mu every week or so, and official releases every few months. You can always download the nightly builds or simply run from source: our master branch is always tested with 100% coverage so even the current development version is fairly stable.